src/forms/form-fields/drop-down-async.js

```diff
@@ -135,19 +140,22 @@ class DropDownAsync extends Component {
             .then((options) => {
                 // Behold the mother of all hacks
                 if (!this.omgLikeJustStop) {
+                 // Behold the very special hack for renaming the very special 'default' cat combo to 'None'
+                    const renamedOpts = options.map(option => Object.assign(
+                        option,
+                        option.model &&
+                        option.model.modelDefinition &&
+                        option.model.modelDefinition.name === 'categoryCombo' &&
+                        option.text === 'default'
```

Comment:
How will this work when the system is being used in another language? I mean `option.text === 'default'`.... Won't `'default'` get translated too?

If so, then you can use this instead:
`option.text === d2i.i18n.getTranslation('default')`

Reply:
This is a hack that was not written by me, I just moved it to be able to return the options from the promise.
Default is a special object name that is not translated as far as I know.